src/engine/meta/meta_blks_mgr.cpp

```diff
@@ -399,6 +400,14 @@ void MetaBlkMgr::register_handler(const meta_sub_type type, const meta_blk_found
     HS_LOG(INFO, metablk, "[type={}] registered with do_crc: {}", type, do_crc);
 }
 
+void MetaBlkMgr::create_sobject(meta_sub_type type) {
+    if(m_sub_info[type].meta_bids.size()==1) {
+        auto sm{HomeStoreBase::safe_instance()->sobject_mgr()};
```

Comment:
nitpick: remove sm and do the call directly as `sm` doesn't seem used anywhere else.